### DISCUSSION TITLE
空白页，如何不算页码（虽然现在不显示，但是页码编码时还是算做一页）？学院现在要求页码必须连续，空白页不能算页码

### DISCUSSION THREAD

**Discussion by chiguokun** [zoam/xmu-thesis-grd#24] [Q&A] [answered]
空白页，如何不算页码（虽然现在不显示，但是页码编码时还是算做一页）？

学院现在要求页码必须连续，空白页不能算页码

**Comment by onns** [comment]
在改

**Reply by chiguokun** [reply]
谢谢大牛！！！

**Comment by onns** [comment]
@zoam 话说正文的clear doublepage是在哪设置的呀？我咋没找到

**Comment by onns** [comment]
https://github.com/zoam/xmu-thesis-grd/commit/bb5c40c1ad0b00de40ab6bdc6891d93dd45847eb

**Comment by zoam** [comment]
我还是不太明白你们想要什么？如果不要这些空白页，只要把 `twoside` 换成 `oneside` 就可以了吧？
你们是想把空白页留着，而且空白页前后的页码还是连续的？

**Comment by onns** [comment]
twocolumn这个变量是干嘛用的？

**Reply by zoam** [reply]
我也不太懂。可能是原来的模板里就有的，似乎可以不要？

**Reply by zoam** [reply]
我试了一下，全删了也没发现会出现问题。我觉得这种不知道干嘛的东西，可以删掉。要真有问题可以再加回来。

**Comment by zoam** [comment]
目前的版本里，空白页也有页眉页码。似乎这样也不太对。谁能说一下目前的标准到底是什么？我被搞懵逼了。

![image](https://user-images.githubusercontent.com/5723558/119322254-0c350780-bcb0-11eb-8bb1-8e222f7a660f.png)

**Reply by onns** [reply]
![Screenshot_2021-05-24-18-07-22-376_com tencent mobileqq](https://user-images.githubusercontent.com/16622934/119332294-f842d300-bcba-11eb-8217-bdd7b8bb2cec.jpg)

**Reply by zoam** [reply]
学院这个标准真是瞎搞。没见过这么土炮的规定。真的怀疑这些人到底懂不懂排版。

**Reply by zoam** [reply]
我在网上找了一个解决方案，可以做到空白页留着，而且空白页前后的页码还是连续的。我 push 上去？

**Reply by onns** [reply]
你发在这我看看

**Reply by onns** [reply]
代码和效果都发一下~因为我下午也百度了，我是改了很久没改出来，因为页眉会有问题

**Reply by zoam** [reply]
认真看了一下，这个方案会导致其他排版问题。还得再研究研究。

**Reply by onns** [reply]
你先别研究了，我快搞出来了，就剩一个bug了。

**Reply by zoam** [reply]
ok

**Reply by onns** [reply]
目前有个问题。。。编译出来正文前面会多一页。。我自己找一个多小时了没找到，你看看你能不能找到（虽然不影响）
[1.tar.gz](https://github.com/zoam/xmu-thesis-grd/files/6532053/1.tar.gz)
![c2918743dd98ce794f4ae582f6b14c6](https://user-images.githubusercontent.com/16622934/119342719-1105b580-bcc8-11eb-880b-02050654018e.png)

**Reply by onns** [reply]
找不到到时候拿pdf编辑器把这个页删掉就行了

**Reply by zoam** [reply]
似乎把这句屏蔽掉就好了。你看看你那边有没有其他问题。

![image](https://user-images.githubusercontent.com/5723558/119347151-b1aaa400-bccd-11eb-85fe-747fc2159009.png)

**Reply by onns** [reply]
我试过，没用

**Reply by onns** [reply]
还能升权限嘛？你帮我创建一个dev分支吧，因为有bug我就不发主分支了

**Reply by zoam** [reply]
你先帮我试一下这个版本，我这边测试好像可以。如果这个版本不行我再建一个新分支。

[xmu-thesis-grd.tar.gz](https://github.com/zoam/xmu-thesis-grd/files/6532494/xmu-thesis-grd.tar.gz)

**Reply by onns** [reply]
没什么问题

**Reply by zoam** [reply]
ok。那我就更新一下 cls 文件。

**Reply by onns** [reply]
等下，你先pull一下

**Reply by onns** [reply]
你的英文目录有问题的

**Reply by onns** [reply]
https://github.com/zoam/xmu-thesis-grd/commit/a37d1a4f6aafeaff4820b0b34f50eb76848f5ba4

**Reply by onns** [reply]
```latex
\newcommand\adjustafterskip{
	\ctexset{chapter={afterskip = 30pt}}
}
```
这个好像没用？

**Reply by onns** [reply]
然后你之前参考文献有加`reducespacing`也不要了嘛
![image](https://user-images.githubusercontent.com/16622934/119353663-bc693700-bcd5-11eb-81cf-ce5b9b06ea2a.png)

**Reply by onns** [reply]
然后cleardoublepage的twocolumn我感觉也可以删除了，这里可以缩进一下方便阅读，别的没问题了

**Reply by zoam** [reply]
哦，我准备是改成要在 demo.tex 里调用一次 `\adjustafterskip`。之前在很多个地方，包括总结那一章分别调 `\reducespacing` 感觉很乱。

**Reply by onns** [reply]
ok，那没别的问题，等你更新，虽然不知道你是咋弄的但是比我优雅哈哈哈哈

**Reply by onns** [reply]
说实话这个要求也恶心到我了，而且还是强制的

**Reply by onns** [reply]
然后我今天下午发的那版本还是保留在历史里吧，因为这样打印完的人可以只替换几页

**Reply by zoam** [reply]
做不到优雅啊。。。太难了。版本要怎么保留？发布 Releases？

**Reply by onns** [reply]
> 做不到优雅啊。。。太难了。版本要怎么保留？发布 Releases？

起码比我优雅
不用，你正常git操作就行，我发现你基本都是在自己的版本上改东西= =正常应该是先pull更新再push自己的代码吧2333

**Reply by zoam** [reply]
被你发现了，哈哈。也没落后太多版本啦。主要是没这个 pull 的习惯。看你的更新记录补上去就好了。

**Reply by chiguokun** [reply]
谢谢大佬，辛苦您了

**Comment by onns** [comment, marked as answer]
https://github.com/zoam/xmu-thesis-grd/commit/2630de081a6d9435e7eae280f486af6fbaff2aad
